tools/clang/test/CMakeLists.txt

```diff
@@ -122,6 +123,7 @@ if (WIN32)
   add_lit_target("check-clang-taef-exec" "Running lit suite hlsl execution test"
             ${CMAKE_CURRENT_SOURCE_DIR}/taef_exec
             PARAMS ${CLANG_TEST_PARAMS}
+                  need_dxil_validator=True
```

Comment:
I don't think this is the right mechanism.  You don't technically need the validator to run execution tests if you use experimental shader models.  If you do want to use a validator, you don't necessarily want a random one from the path.  You want to use one specifically selected for testing.  In either case, we don't want a random DXIL.dll from the path to be used.